glommio/src/sys/uring.rs

```diff
@@ -92,7 +92,7 @@ enum UringOpDescriptor {
 }
 
 #[derive(Debug)]
-struct UringDescriptor {
+pub(crate) struct UringDescriptor {
```

Comment:
I don't see where this is being used, but can we make do with `pub(super)`?

Reply:
`UringDescriptor` is now part of the `UringCommin` trait and since that one is `pub(crate)`, this one needs to be as well. 
At some poi not it would be nice to break that trait apart